isamples_metadata/GEOMETransformer.py

```diff
@@ -205,9 +205,9 @@ def keywords(self) -> typing.List[str]:
         return keywords
 
     def produced_by_id_string(self) -> str:
-        parent_record = self._source_record_parent_record()
-        if parent_record is not None:
-            return parent_record["bcid"]
+        main_record = self._source_record_main_record()
+        if main_record is not None:
+            return main_record["bcid"]
```

Comment:
This was a bug in the original implementation that no one ever noticed -- this always referred to the event rather than the parent record.